test/org/lice/parse2/LexIdentifierTest.java

```diff
@@ -0,0 +1,40 @@
+package org.lice.parse2;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+
+public class LexIdentifierTest {
+	@Test(timeout = 100)
+	public void testLexIdentifier() {
+		String srcCode = "Fuck@dentifier";
+		Lexer l = new Lexer(srcCode);
+		assertTrue(l.currentToken().getKind() == Token.TokenType.Identifier);
+		assertTrue(l.currentToken().getStrValue().equals("Fuck@dentifier"));
+		assertTrue(l.peekOneToken().getKind() == Token.TokenType.EOI);
+	}
+
+	@Test(timeout = 100)
+	public void testLexIdentifiers() {
+		String srcCode = "Fuck@dentifier _Yet1A2not->her@dentifier";
+		Lexer l = new Lexer(srcCode);
+		assertTrue(l.currentToken().getKind() == Token.TokenType.Identifier);
+		assertTrue(l.currentToken().getStrValue().equals("Fuck@dentifier"));
+		l.nextToken();
+		assertTrue(l.currentToken().getKind() == Token.TokenType.Identifier);
+		assertTrue(l.currentToken().getStrValue().equals("_Yet1A2not->her@dentifier"));
+		assertTrue(l.peekOneToken().getKind() == Token.TokenType.EOI);
+	}
+
+	@Test(timeout = 100)
+	public void testLexIdentifiers2() {
+		String srcCode = "Fuck@dentifier,_Yet1A2not->her@dentifier";
+		Lexer l = new Lexer(srcCode);
+		assertTrue(l.currentToken().getKind() == Token.TokenType.Identifier);
+		assertTrue(l.currentToken().getStrValue().equals("Fuck@dentifier"));
+		l.nextToken();
+		assertTrue(l.currentToken().getKind() == Token.TokenType.Identifier);
+		assertTrue(l.currentToken().getStrValue().equals("_Yet1A2not->her@dentifier"));
+		assertTrue(l.peekOneToken().getKind() == Token.TokenType.EOI);
```

Comment:
Use `assertEquals`.

Reply:
Switched to assertEquals in commit https://github.com/lice-lang/lice-tiny/pull/1/commits/a159cfb618965588b171b2d348c7e0a3e5ff68be .

Reply:
Good!